pkgs/development/interpreters/clojure/babashka.nix

```diff
@@ -23,6 +23,9 @@ stdenv.mkDerivation rec {
 
     # https://github.com/babashka/babashka/blob/v0.6.1/script/compile#L41-L52
     args=("-jar" "$BABASHKA_JAR"
+          # Reduce closure size by linking using glibc directly
+          # instead of using GraalVM copies
+          "-H:CLibraryPath=${glibc}/lib"
```

Comment:
Maybe a better a approach here would be to modify the `graalvm-ce` derivation to have a `dev` output and put those glibc libraries there instead. This would make it easier on the future if some new libraries were added to GraalVM.

However, I couldn't make it to work. I think the GraalVM derivation is currently broken, since any modification (just sufficient to avoid the current GraalVM version from Hydra cache) makes the build break.

Reply:
However, just by adding the `outputs = [ "out" "dev" ];` on GraalVM derivation breaks it, so I don't know what I am doing wrong.

Reply:
Found what I should do, it needs to be `lib` not `dev`.